### ISSUE TITLE
Select Media from Server

### ISSUE THREAD

**Issue by krebbi** [lucasnetau/formBuilder-plugin-media#1]
First of all thanks for your Plugin! 

Is it possible to populate Thumbnails from the Server and the user is able to select one of them?

**Comment by lucasnetau** [comment]
Hi @krebbi , thanks for giving it a go.

So you would like to allow the plugin to read a directory from a server and allow the selection of an already uploaded image, rather than uploading an image directly?

**Comment by krebbi** [comment]
Yes, I would like to give the plugin an array of urls from which the user can select an element to insert.

**Comment by johnpg82** [comment]
@lucasnetau Primarily the problem is if you upload a file that is too large if it is not server side it consumes all the memory in the browser and causes the browser to crash. Sending that data server side will remedy this issue.

**Comment by h3786010** [comment]
simple way:
using $.ajax() and FormData() instead of FileReader()
then return URL string after uploaded file.

**Comment by johnpg82** [comment]
Yeah I got it to work that way. Here is an example.
https://github.com/johnpg82/formBuilder-plugin-media/blob/Server-Side-Upload/media.js
I ended up just using upload.io as it offered cropping and some other features I wanted.

**Comment by lucasnetau** [comment]
> simple way: using $.ajax() and FormData() instead of FileReader() then return URL string after uploaded file.

Yes, if you disable the default handler, then you can use whatever method you wish to upload. If you have any examples you want to included I can add them to the README. 

There are many ways to handle the upload and post processing, depending on your use case and security model. Upload to AWS S3 with post processing is another method I've used.

For uploads in the field from short videos, sound recordings and photos we have found the data URI method to be quick and painless
